common/flavorization/wc_title_holders.txt

```diff
@@ -121,4 +121,65 @@ overlord_female = {
 	top_liege = no
 	culture_groups = { orc_group }
 	governments = { feudal_government clan_government }
+}
+
+### Nightborne
+# Emperor
+king_nightborne_male = {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = 201
+	top_liege = no
+	cultures = { nightborne }
+}
+queen_nightborne_female = {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = 201
+	top_liege = no
+	cultures = { nightborne }
+}
+
+# King
+prince_nightborne_male = {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = 146
+	top_liege = no
+	cultures = { nightborne }
+}
+princess_nightborne_female = {
+	type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = 146
+	top_liege = no
+	cultures = { nightborne }
+}
+
+# Duke
+lord_nightborne_male = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 126
+	top_liege = no
+	cultures = { nightborne }
+}
+lady_nightborne_female = {
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 126
+	top_liege = no
+	cultures = { nightborne }
 }
```

Comment:
Do we want the title to be called Lordship?

Reply:
The general opinion of the dev dc channel seemed to be that Lordship was too generic